kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/config/S3Config.scala

```diff
@@ -108,6 +102,16 @@ object Format extends Enum[Format] {
 
   case object Bytes extends Format
 
+
+  def apply(format: String): Format = {
```

Comment:
enumeratum already defines a function to read and returnf from string.

Reply:
It's used 3 lines below
Format.withNameInsensitiveOption()

This function serves to add a specific error message and to supply a default when no format has been defined.